tf2_geometry_msgs/test/test_tf2_geometry_msgs.cpp

```diff
@@ -67,6 +67,46 @@ geometry_msgs::msg::TransformStamped generate_stamped_transform()
   return t;
 }
 
+// Helpers used by gtest to print quaternions when errors happen
+// Must be defined in the namespace the class to be printed in lives, as per
+// https://github.com/google/googletest/blob/main/docs/advanced.md#teaching-googletest-how-to-print-your-values
+namespace geometry_msgs::msg
+{
+std::ostream& operator<<(std::ostream& stream, const Quaternion & q)
```

Comment:
```suggestion
std::ostream & operator<<(std::ostream & stream, const Quaternion & q)
```

I think this is the answer uscrustify is looking for.